doc/users/next_whats_new/2017_12_17_compare_colors.rst

```diff
@@ -0,0 +1,4 @@
+Comparision of 2 colors in Matplotlib
+-------------------------------------
+
+As Matplotlib comes with a wide variety of colors, therefore this new version allows 2 legend colors to be compared. 
```

Comment:
- The sentence doesn't quite sound right - I would just remove 'therefore'
- As @jklymak , could you reference the actual method by adding

```
`~matplotlib.colors.same_color`
```
This will show up as a link that says "same_color"

Reply:
A style thing, but I somewhat favour the full names so that users can see what modules routines come from?

Reply:
Yeah, that's a good point, if you could use this instead:

```
`matplotlib.colors.same_color`
```